src/universal/decorators/socketWithPresence/presenceSubscriber.js

```diff
@@ -0,0 +1,32 @@
+import subscriptions from 'universal/subscriptions/subscriptions';
+import socketCluster from 'socketcluster-client';
+import {PRESENT, SOUNDOFF, LEAVE} from 'universal/subscriptions/constants';
+import {cashay} from 'cashay';
+
+export default function presenceSubscriber(subscriptionString, variables, handlers) {
+  const {channelfy} = subscriptions.find(sub => sub.string === subscriptionString);
+  const channelName = channelfy(variables);
+  const socket = socketCluster.connect();
+  const {add, remove} = handlers;
+  socket.subscribe(channelName, {waitForAuth: true});
+  socket.watch(channelName, data => {
+    if (data.type === SOUNDOFF) {
+      const options = {
+        variables: {
+          meetingId: variables.meetingId,
+          targetId: data.targetId
+        }
+      };
+      cashay.mutate('present', options);
```

Comment:
If someone joins, they ask everyone to identify themselves by calling `SOUNDOFF`. Folks reply by saying `present`.